sample-app/build.gradle

```diff
@@ -70,6 +70,12 @@ dependencies {
     testImplementation 'junit:junit:4.12'
     androidTestImplementation 'com.android.support.test:runner:1.0.1'
     androidTestImplementation 'com.android.support.test.espresso:espresso-core:3.0.1'
+
+    debugImplementation 'com.squareup.leakcanary:leakcanary-android:1.6.2'
+    releaseImplementation 'com.squareup.leakcanary:leakcanary-android-no-op:1.6.2'
```

Comment:
I think there's no need for no-op for release as this is a sample app, we would want to detect leaks event in release builds

Reply:
you need to change it to `implementation` without specifying release/debug - probably CI is failing now as release cannot be built.

Reply:
fixed now